category-service/pom.xml

```diff
@@ -82,6 +82,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
```

Comment:
Why adding h2 database? We are already using Postgres. Please remove if not necessary.